package.json

```diff
@@ -8,5 +8,8 @@
   "repository": "https://github.com/enactjs/agate-apps",
   "eslintConfig": {
     "extends": "enact/strict"
+  },
+  "dependencies": {
+    "roslib": "^0.20.0"
```

Comment:
Is `roslib` a dependency of the "components" folder components? I didn't think so. This may need to be removed.